### PR TITLE
Updated refresh in

### PR DESCRIPTION
**Pull request by Veena11**
Describe what this PR is trying to do.

## Proposed changes
“refresh_in” will be communicated to the client library (MSAL) as part of the token response for the following grant_types supported by MSAL Obj-C:
o authorization_code
o refresh_token
• this will be added as a time in seconds at which MSAL should attempt to refresh the token
• Refresh Attempt failure:
o If an attempt to refresh an unexpired token fails, then simply return the existing token
o If an attempt to refresh an expired token fails as a result of service being unavailable, then return extended lifetime access token if extended lifetime access enabled. (No change in current behavior)

## Type of change

- [x ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [x] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: IdentityCore/src/MSIDOAuth2Constants.h [hunk 1]
```diff
@@ -37,6 +37,7 @@ extern NSString *const MSID_OAUTH2_ERROR;
 extern NSString *const MSID_OAUTH2_ERROR_DESCRIPTION;
```
Comment (kaisong1990) [inline IdentityCore/src/MSIDOAuth2Constants.h:37]: This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.

Comment (kaisong1990) [reply IdentityCore/src/MSIDOAuth2Constants.h:37]: This is auto generated message that whenever a PR is created and no change logs is updated, so feel free to ignore it if needed

Comment (kaisong1990) [reply IdentityCore/src/MSIDOAuth2Constants.h:37]: Also I would recommend to create a draft pr if was not ready for review :)

**Comment by kaisong1990** [comment]
@Veena11 Could we close this one if not needed :)

**Comment by Veena11** [comment]
Duplicate PR https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/975  , hence closing this PR

**Pull request closed without merging**